controllers/apps/transformer_component_account_provision.go

```diff
@@ -88,6 +89,10 @@ func (t *componentAccountProvisionTransformer) Transform(ctx graph.TransformCont
 		return nil
 	}
 	for _, account := range transCtx.SynthesizeComponent.SystemAccounts {
+		// The secret of initAccount should be rendered into the config file.
```

Comment:
"The secret of initAccount should be rendered into the config file."  --> "The secret of initAccount should be rendered into the config file via environment variables, or injected into the container through specific account&password environment variables name supported by the engine."